amnestywebsite/humanity-theme
# Post List - Link List Custom Background Image

Ref: https://github.com/amnestywebsite/humanity-theme/issues/278

Adds the ability to add a background image to the Post List block when using Link List Style and Custom Type

**Steps to test**:
1. Edit a post
2. Add a Post List block
3. Ensure "Style" is set to "Link List" & "Type" is set to "Custom"
4. You should see the image icon in the top right now
5. Add an image and populate with content
6. The background should change to the selected image in the editor
7. Now save and check the front end for the same

Example: http://bigbite.im/v/OoxiC6
